src/main/java/com/gildedrose/aplication/UpdateDailyValues.java

```diff
@@ -0,0 +1,41 @@
+package com.gildedrose.aplication;
+
+import com.gildedrose.domain.Item;
+import com.gildedrose.infraestructure.ItemsService;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+public class UpdateDailyValues {
+
+    public ItemsService itemsService = new ItemsService();
+    public List<Item> items;
+    public final Map<String, Consumer<Item>> updateMethods;
+    public UpdateDailyValues(List<Item>items) {
+        this.items = items;
+        this.updateMethods = Map.of(
+                "Aged Brie", itemsService::dailyUpdateAged,
+                "Sulfuras, Hand of Ragnaros", item -> {},
+                "Conjured Mana Cake", itemsService::dailyUpdateConjured,
```

Comment:
Hace poco contratamos a un proveedor de artículos conjurados mágicamente. Esto requiere una actualización del sistema:
Tras la actualización del sistema TODOS los items tienen que poder ser Conjured o no.... todos.

Reply:
We have recently signed a supplier of conjured items. This requires an update to our system:
Hace poco contratamos a un proveedor de artículos conjurados mágicamente. Esto requiere una actualización del sistema:
La actualizacion es sencilla añadir u nuevo tipo de objeto Conjured, en ningun caso dice que los items con otras caracteristicas concretas vayan a ser tambien conjured, del mismo modo que un item de tipo Sulfuras no va a ser tb item con caracteristicas Aged.

Esta aplicacion no esta tratando un objeto con varias caracteristicas, trata objetos que segun su nombre tienen una caracteristica u otra.

En el supuesto de que con la Clase item tal cual esta (Que esta fatal hecha, por cierto) quisiera que tendrian varias caracteristicas que se incluirian en el propio nombre ya que no hay otra forma de identificar los items, el nombre deberia ser un String y recorrerlo en busca de las palabras clave para saber cuantas caracteristicas tiene ese item y actuar en consecuencia con el aumento/disminucion de valores.
Ejemplo: nombre = "Sulfuras, conjured, aged brie casco de la locura."
Y modificar los metodos para que hagan esta busqueda.

Reply:
El enunciado es muy ambiguo , se puede interpretar de varias maneras.

Reply:
Voy a dar la solucion asi y cuando lo tengamos lo hago modificando la clase item para no liarnos tanto.